ci_environment/mysql/templates/default/my.cnf.erb

```diff
@@ -51,14 +51,17 @@ basedir         = /usr
 datadir         = <%= node['mysql']['data_dir'] %>
 tmpdir          = /tmp
 skip-external-locking
+
 #
 # Instead of skip-networking the default is now to listen only on
 # localhost which is more compatible and is not less secure.
 bind-address            = <%= node['mysql']['bind_address'] %>
 
+innodb_file_per_table  = true
+innodb_file_format     = Barracuda
```

Comment:
`innodb_large_prefix` requires Barracuda. I didn't dig into the possible others advantage of the new format but there should be no regression from Antelope.